src/shell/shell.cpp

```diff
@@ -1258,6 +1260,31 @@ void SHELL_Init() {
 	        " Volume in drive %c is %s\n"
 	        " Volume Serial Number is %04X-%04X\n"
 	        "\n");
+	MSG_Add("SHELL_CMD_MOVE_HELP",
+	        "Moves files and renames files and directories\n");
+	MSG_Add("SHELL_CMD_MOVE_HELP_LONG",
+	        "Usage:\n"
+	        "To move one or more files:\n"
+	        "  [color=green]move[reset] [color=cyan][drive:][path]filename1[,...] destination[reset]\n"
+	        "\n"
+	        "To rename a directory:\n"
```

Comment:
Remove `To move one or more files:\n` and `To rename a directory:\n` parts - if explanation is needed, put it in `Notes:` section.

Reply:
Yes, and I think we already have enough explanation and examples in the provided help (🎉 for that one! 😄), so this is redundant and just uses up valuable vertical space.

Reply:
Fixed